packages/search-common/package.json

```diff
@@ -10,6 +10,9 @@
     "main": "dist/index.cjs.js",
     "types": "dist/index.d.ts"
   },
+  "backstage": {
+    "role": "node-library"
```

Comment:
This package actually only contains types at the moment, and ends up being included in both frontend and backend.

Reply:
It's because it doesn't output `esm`, but let's get that fixed instead. I switched it over.